endpoint/endpoint.go

```diff
@@ -17,7 +17,8 @@ limitations under the License.
 package endpoint
 
 const (
-	ownerKey = "owner"
+	// OwnerKey Labels key for storing owner id
+	OwnerKey = "owner"
```

Comment:
Let's change it to the following to be more explaing and similar to the stuff in `source/source.go`:

```go
// OwnerLabelKey is the name of the label that defines the owner of an Endpoint.
OwnerLabelKey = "owner"
```